examples/ConsoleApp/FeatureFilters/AccountIdFilter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Consoto.Banking.AccountService.FeatureFilters;
+using Microsoft.Extensions.Configuration;
+using Microsoft.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Consoto.Banking.AccountService.FeatureManagement
+{
+    /// <summary>
+    /// A filter that uses the feature management context to ensure that the current task has the notion of an account id, and that the account id is allowed.
+    /// This filter will only be executed if an object implementing <see cref="IAccountContext"/> is passed in during feature evaluation.
+    /// </summary>
+    [FilterAlias("AccountId")]
+    class AccountIdFilter : IContextualFeatureFilter<IAccountContext>
+    {
+        public Task<bool> EvaluateAsync(FeatureFilterEvaluationContext featureEvaluationContext, IAccountContext accountId)
```

Comment:
>accountId [](start = 113, length = 9)

accountContext?

Reply:
Yes, thanks.

---
In reply to: [328363265](https://github.com/microsoft/FeatureManagement-Dotnet/pull/14#discussion_r328363265) [](ancestors = 328363265)

Reply:
this one

---
In reply to: [328746974](https://github.com/microsoft/FeatureManagement-Dotnet/pull/14#discussion_r328746974) [](ancestors = 328746974,328363265)